### ISSUE TITLE
Create fluent API for JsonValueReaderWriter on a property

### ISSUE THREAD

**Issue by ajcvickers** [dotnet/efcore#31647]
Split out from #30677

**Comment by qihangnet** [comment]
This feature is too important for users who use EF+ PostgreSQL. For example, some of our JSON data is written in other programming languages, and they follow the SnakeCase JSON format.

**Comment by roji** [comment]
@qihangnet I don't think this issue would cover that - you're likely looking for https://github.com/efcore/EFCore.NamingConventions/issues/269.

**Comment by qihangnet** [comment]
@roji Yes, sorry, I got them mixed up when I was reading multiple issues at the time.
